### DISCUSSION TITLE
How to use aquacrop-7.1-x86_64-windows

### DISCUSSION THREAD

**Discussion by llq4399** [KUL-RSDA/AquaCrop#353] [Q&A] [answered]
I would like to know how people use the standalone version of Aquacrop. I want to know how to call it and batch processing. The first thing I encountered was the GUI version. Then found AquacropOnR, wanted to use it to call aquacrop, and perform Morris sensitivity analysis. It was later discovered that AquacropOnR only works with standalone versions of Aquacrop. So I downloaded aquacrop-7.1-x86_64-windows. But in the using process ,  I found that I could not resolve the conflict between scenario setting and reading PRO files.
> source("~/.active-rstudio-document")
# A tibble: 1 × 12
  Scenario Input_Date Sim_Date   Plant_Date IRRI  Soil  Plu   Tnx   ETo  
  <chr>    <date>     <date>     <date>     <chr> <chr> <chr> <chr> <chr>
1 YXX2015… 2015-01-01 2015-01-01 2015-01-01 Rain… D:/R… D:/R… D:/R… D:/R…
# ℹ 3 more variables: FMAN <chr>, GWT <dbl>, SW0 <chr>
Error in `map()`:
ℹ In index: 1.
ℹ With name: Scenario.
Caused by error in `file()`:
! 无法打开链结
Run `rlang::last_trace()` to see where the error occurred.
警告信息:
In file(file, ifelse(append, "a", "w")) :
  无法打开文件'LIST/YXX2015CZ.PRO': No such file or directory

**Comment by gdelannoy** [comment, marked as answer]
AquacropOnR is a wrapper in R, just like there is a wrapper in Python (https://github.com/KUL-RSDA/RegionalAC_Py). We only maintain the latter and refer to the developers of AquacropOnR for the former.

Instructions on how to run the Standalone of AquaCrop in Windows are here:
https://github.com/KUL-RSDA/AquaCrop/discussions/323
